rcjaRegistration/invoices/models.py

```diff
@@ -326,6 +326,15 @@ def amountDueInclGST(self):
         return round(self.invoiceAmountInclGST() - self.amountPaid(), 2)
     amountDueInclGST.short_description = 'Amount due (incl GST)'
 
+    def amountDuePaypal(self):
+        if self.amountDueInclGST() < 0.05: # 0.05 to avoid tiny sum edge caes
```

Comment:
This seems fine but important to be careful with allowing thresholding like this for obvious reasons

Reply:
Yes it isn't ideal. The entire dealing of floats as currently done isn't ideal, but in reality probably doesn't matter.

Also worth noting that the amountDue will still show the 5 cents and it will be visible in the admin and export, so you wouldn't really get away with it.